site/docs/4.6.0/overview/releaseNotes.md

```diff
@@ -2,16 +2,35 @@
 title: Apache BookKeeper 4.6.0 Release Notes
 ---
 
-[provide a summary of this release]
+This is the seventh release of BookKeeper as an Apache Top Level Project!
+
+The 4.6.0 release incorporates new fixes, improvements, and features since previous major release 4.5.0.
 
 Apache BookKeeper users are encouraged to upgrade to 4.6.0. The technical details of this release are summarized
 below.
 
 ## Highlights
 
-[List the highlights]
+The main features in 4.6.0 cover are around following areas:
+
+- Persistable bookie status
+- BookKeeper Admin REST API
+- New CreateLedger API
+- lifecycle components for managing components in bookie server
+- Introduce write FileInfo cache and read FileInfo cache
+- Use ByteBuf for entrylogger reads
```

Comment:
Not sure we need to call this out separately. It should be one part of performance improvement.